russian/call-mom/call-back/call-back.md

```diff
@@ -0,0 +1 @@
+Вы снова набираете номер, однако механический голос сообщает - "4815162342 Данный номер не обслуживается. Авто-система связи станции Лебедь" 
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.